OneCommunityGlobal/HighestGoodNetworkApp
# Demi_tools_list_default_view

# Description
Implements # 4.1.1 on Phase 2 WBS

## Related PRS (if any):
None

## Main changes explained:
- Created a new page for "Tools List" to be displayed

## How to test:

1. git pull
2. check into current branch
3. npm install and npm run start:local to run this PR locally
4. Clear site data/cache in local storage
5. log in as any user
6. go to http://localhost:3000/bmdashboard/login to confirm your login credentials
7. go to http://localhost:3000/bmdashboard/tools

## Screenshots or videos of changes:

## Note:
Use development branch for the back end.
